crates/torii/core/src/engine.rs

```diff
@@ -610,6 +610,8 @@ impl<P: Provider + Send + Sync + std::fmt::Debug + 'static> Engine<P> {
         let mut unique_contracts = HashSet::new();
         // Contract -> Cursor
         for (event_idx, event) in events.iter().enumerate() {
+            // NOTE: erc* processors expect the event_id to be in this format to get
+            // transaction_hash:
             let event_id =
                 format!("{:#064x}:{:#x}:{:#04x}", block_number, transaction_hash, event_idx);
```

Comment:
Couldn't we have a function that encapsulates that and add the `NOTE` as doc? :)

Reply:
i think we still need the Note there so if someone wants to change the event_id structure they know about this dependency